sdk/storage/azure-storage-queue/azure/storage/queue/_models.py

```diff
@@ -275,9 +277,16 @@ def __init__(self, command, results_per_page=None, continuation_token=None):
         )
         self._command = command
         self.results_per_page = results_per_page
+        self._max_messages = max_messages
```

Comment:
Nit: It feels weird to me to be modifying a member named `max_messages`. Perhaps the name of this member should be changed to something like `_message_count` or something that better reflects that this is actually a count of the remaining messages we want to grab, and it just starts at `max_messages`? (It's only max_messages in the beginning).

This is a nit though so feel free to WF :)

Reply:
For once I will actually take the opposite stance- I think `max_messages` is okay since it starts at the maximum number of messages we want to receive, and then is decreasing in value. Something like `message_count` in my eyes makes sense if we were starting at 0 and keeping track of messages we've received (i.e. increasing in value). So I will leave it for now, although always open to change if we have others chime in that feel strongly one way or the other 😃